.github/workflows/ci.yml

```diff
@@ -362,6 +362,8 @@ jobs:
 
   event_file:
     name: "Event File"
+    needs: build-java
+    if: always() && needs.build-java.result != 'skipped'
```

Comment:
I think that `needs: build-java` would be enough to skip the job when `build-java` was skipped itself

Reply:
not if `build-java` fails due to test failures, then `result` is `failure`